src/sentry/api/bases/organization.py

```diff
@@ -276,8 +276,16 @@ def get_filter_params(
         try:
             start, end = get_date_range_from_params(request.GET, optional=date_filter_optional)
             if start and end:
-                with configure_scope() as scope:
-                    scope.set_tag("query.period", (end - start).total_seconds())
+                total_seconds = (end - start).total_seconds()
+                sentry_sdk.set_tag("query.period", total_seconds)
+                sentry_sdk.set_tag(
+                    "query.period.grouped",
+                    "<=24h"
+                    if total_seconds <= 86400
+                    else "<=30d"
```

Comment:
Do we want to add in steps for 1h, 7d, and 14d?

Reply:
Hmm 1h seems too short, but I'll add for 7d + 14 :+1: